listenbrainz/webserver/static_manager.py

```diff
@@ -16,4 +16,4 @@ def read_manifest():
 def get_static_path(resource_name):
     if resource_name not in manifest_content:
         return "/static/%s" % resource_name
-    return "/static/js/dist/%s" % manifest_content[resource_name]
+    return manifest_content[resource_name]
```

Comment:
before, the manifest file only contained the basename, but now it includes the full path?

Reply:
Correct. That's from the addition of output.publicPath in the webpack config.

Looks like with this new version you have to have a public path defined (maybe it would work with an empty string, haven't trie) or it defaults to "auto", so you get something like `"recentListens.js":"autorecentListens.5ab3956377725756fbf5.js"`, note the "auto" at the beginning of the file path.

It also seemed to me that it's better to defined it in one spot rather than two, and the webpack config is that place for that.